server/tests/e2e/user.e2e-spec.ts

```diff
@@ -1200,6 +1202,157 @@ describe('User', () => {
         });
     });
 
+    describe('GET /api/user/maps/submitted', () => {
+        const expects = (res) => expect(res.body).toBeValidPagedDto(MapDto);
+
+        it('should retrieve the list of maps that the user submitted', async () => {
+            const res = await get({
+                url: 'user/maps/submitted',
+                status: 200,
+                token: user1Token
+            });
+
+            expects(res);
+            expect(res.body.totalCount).toBe(2);
+            expect(res.body.returnCount).toBe(2);
+            expect(res.body.response[0].id).toBe(map3.id);
+            expect(res.body.response[1].id).toBe(map1.id);
+        });
+
+        it('should retrieve an empty map list', async () => {
+            const res = await get({
+                url: 'user/maps/submitted',
+                status: 200,
+                token: user2Token
+            });
+
+            expect(res.body.totalCount).toBe(0);
+            expect(res.body.returnCount).toBe(0);
+        });
+
+        it('should retrieve the users submitted maps when using the skip query parameter', () =>
+            skipTest({
+                url: 'user/maps/submitted',
+                test: expects,
+                token: user1Token
+            }));
+
+        it('should retrieve the users submitted maps when using the take query parameter', () =>
+            takeTest({
+                url: 'user/maps/submitted',
+                test: expects,
+                token: user1Token
+            }));
+
+        it('should retrieve the submitted maps with expanded info', async () => {
+            const prisma: PrismaService = global.prisma;
+
+            await prisma.map.update({
+                where: {
+                    id: map2.id
+                },
+                data: {
+                    info: {
+                        create: {
+                            description: 'Second map',
+                            numTracks: 1,
+                            creationDate: new Date()
+                        }
+                    }
+                }
+            });
+
+            await prisma.map.update({
+                where: {
+                    id: map3.id
+                },
+                data: {
+                    info: {
+                        create: {
+                            description: 'Third map',
+                            numTracks: 1,
+                            creationDate: new Date()
+                        }
+                    }
+                }
+            });
```

Comment:
This works but we may as well always add the mapinfo stuff when we create `map2` and `map3`, same as how `map1` is set up,